arbitrum-docs/welcome/get-started.md

```diff
@@ -0,0 +1,80 @@
+| [Arbitrum Nitro](/inside-arbitrum-nitro)                          | The node **software** that codifies the Rollup and AnyTrust protocols.                              |
+| [Arbitrum nodes](/node-running/how-tos/running-a-full-node)       | **Machines** that run Nitro in order to service and/or interact with an Arbitrum chain.             |
+| [Arbitrum One](https://portal.arbitrum.io/?chains=arbitrum-one)   | A public Rollup **chain**.                                                                          |
+| [Arbitrum Nova](https://portal.arbitrum.io/?chains=arbitrum-nova) | A public AnyTrust **chain**.                                                                        |
+| [Arbitrum bridge](https://bridge.arbitrum.io/)                    | Lets you move ERC-20 tokens between Ethereum and Arbitrum.                                          |
+| [Arbitrum Orbit](https://orbit.arbitrum.io/)                      | Lets you run your own Rollup and AnyTrust chains.                                                   |
+| [Stylus](/stylus/stylus-gentle-introduction)                      | Lets you write EVM-compatible smart contracts in Rust and any other language that compiles to Wasm. |
+
+
+
+## Arbitrum for users
+
+**Users** interact with Arbitrum either through the Arbitrum bridge or by using dApps that have been deployed to an Arbitrum chain.
+
+| Resource                                       | Description                                                     |
+| ---------------------------------------------- | --------------------------------------------------------------- |
+| [Arbitrum bridge](https://bridge.arbitrum.io/) | Lets you move ERC-20 tokens between Ethereum and Arbitrum.      |
+| [Arbitrum Portal](https://portal.arbitrum.io/) | A directory of dApps on Arbitrum.                               |
+| [Quickstart (bridge)](/getting-started-users)  | Provides step-by-step instructions for first-time bridge users. |
+
+
+## Arbitrum for developers
+
+**Developers** build Arbitrum dApps by deploying smart contracts to an Arbitrum chain.
+
+| Resource                                                                 | Description                                                                                              |
+| ------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------- |
+| [A gentle introduction to Arbitrum](/for-devs/gentle-introduction-dapps) | A technical introduction to Arbitrum's suite of scaling solutions.                                       |
+| [Quickstart (Solidity)](/for-devs/quickstart-solidity-hardhat)           | Targeted at web2 developers who want to deploy their first Solidity smart contract to Arbitrum.          |
```

Comment:
I wouldn't emphasize web2 and web3 developers (I would just mention "developers"), but maybe that's just me. Not 100% confident, haha

Reply:
The explicit audience targeting here was intentional :)